ux.symfony.com/templates/icons/index.html.twig

```diff
@@ -0,0 +1,63 @@
+{% extends 'base.html.twig' %}
+
+{% block javascripts %}
+    {{ parent() }}
+    <meta name="turbo-prefetch" content="false">
```

Comment:
The bandwitch coast would be incredibely high, and it would lower all the site (Turbo does not keep in cache the prefetched page with InstantClick ... 😵‍💫 ..) 

And as your mouse will pobably hover 40 icons before you click one....

Reply:
I prefer to stay quiet about this feature and the way it has been decided/implemented :|

Reply:
Yep ! 

That'd need a bit more than that because of the Modal is in the footer.. could we see that in parralel of the numerius bug fixes we'll have / features to add / etc ?